pyproject.toml

```diff
@@ -24,3 +24,6 @@ keywords = ["steel design", "civil engineering", "engineering"]
 
 [project.gui-scripts]
 osdag = "osdag.osdagMainPage:do_stuff"
+
+[project.scripts]
+osdag-cli = "osdag_cli:testing"
```

Comment:
Please keep `osdag_cli` under the `osdag` module (so it should read `osdag.osdag_cli:testing`). Also, change name of the method to something less temporary. Can `main` do the job?